WordPress/Classes/ViewRelated/Activity/ActivityListViewController.swift

```diff
@@ -29,16 +34,54 @@ class ActivityListViewController: UITableViewController, ImmuTablePresenter {
         }
     }
 
+    private enum Constants {
+        static let estimatedRowHeight: CGFloat = 62
+    }
+
     // MARK: - GUI
 
     fileprivate var noResultsViewController: NoResultsViewController?
 
     // MARK: - Constructors
 
-    init(siteID: Int, service: ActivityServiceRemote) {
-        self.siteID = siteID
-        self.service = service
+    init(site: JetpackSiteRef, store: ActivityStore) {
+        self.site = site
+        self.store = store
+
+        self.activitiesReceipt = store.query(.activities(site: site))
+        self.restoreStatusReceipt = store.query(.restoreStatus(site: site))
+
         super.init(style: .plain)
+
+        self.changeReceipt = store.onChange { [weak self] in
+            self?.updateViewModel()
+        }
+
+        self.actionsReceipt = ActionDispatcher.global.subscribe { [weak self] action in
+            switch action {
+            case ActivityAction.rewindStarted(let site, _):
+                guard site == self?.site else { return }
+                self?.showRestoringMessage(0)
+            case ActivityAction.rewindFinished(let site, _):
+                guard site == self?.site else { return }
+                self?.restoreCompleted()
+            case ActivityAction.rewindFailed(let site, _), ActivityAction.rewindRequestFailed(let site, _):
+                guard site == self?.site else { return }
+                self?.restoreFailed()
+            case ActivityAction.rewindStatusUpdateTimedOut(let site):
+                guard site == self?.site else { return }
```

Comment:
(Incoming Unsolicited Feedback!!). @jklausa the standard we've got for the `guard else { }` construct is with the `return` in a new line, always, no matter if there's just a single instruction within the `{ }`

[More here](https://github.com/wordpress-mobile/swift-style-guide#braces)

Reply:
+ In other spots you do use that style ([here](https://github.com/wordpress-mobile/WordPress-iOS/pull/9656/files#diff-73c4f89094df64f0ed783268b6b19e7bR269)), it'd be neat if it would all be consistent!